cores/esp32/esp32-hal-gpio.c

```diff
@@ -208,7 +209,11 @@ static void IRAM_ATTR __onPinInterrupt(void *arg)
         do {
             if(gpio_intr_status_l & ((uint32_t)1 << pin)) {
                 if(__pinInterruptHandlers[pin]) {
-                    __pinInterruptHandlers[pin]();
+					if(arg){
```

Comment:
It looks too much indentation in here. Are there tabs or gitub change like this?